### DISCUSSION TITLE
Unteres und oberes Leistungslimit per Mqtt ändern

### DISCUSSION THREAD

**Discussion by frankydee1980** [hoylabs/OpenDTU-OnBattery#427] [Ideas]
Hallo Zusammen,

nur als Idee. Mir ist aufgefallen, wenn die Akkus voll sind und in den Float Modus gehen, der Victron die Solarleistung runterregelt.
Mein oberes Leistungslimit ist moderat auf 150W eingestellt, damit es auch über die Nacht reicht, aber mein Grundverbrauch ist etwas höher. Wenn der Akku/Victron also in den Float Modus geht wäre es doch gut, das Leistungslimit zur erhöhen. Könnte man über ein Blockly Script in IObroker sicher realisieren. Wenn die Sonne nachläßt, das er dann wieder auf den Grundwert zurückgeht.

Was meint ihr?

Gruß Frank

**Comment by spcqike** [comment]
gibt es dafür nicht Mode 2?

https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/MQTT

> Three modes are implemented:
>     0 - Normal operation: The Power Limiter works as configured through the web application.
>     1 - Fully disable: The inverter is shut down and the Power Limiter stops operating, as if it was disabled in the web application.
>     2 - Unconditional Solar Passthrough: The power limit is set such that all solar power (adjusted for efficiency) is fed into the home. This is done irrespective of grid consumption and without consuming energy from the battery, which is the key difference to the regular Power Limiter Solar Passthrouh feature. Essentially, the inverter mimics the behavior of a traditional, non-smart inverter. This can be particularly useful in scenarios where solar power is better stored elsewhere, such as in an electric car.


Nachtrag:
Damit erhöchst du halt nicht das Leistungslimit auf einen Wert X oder auf den aktuellen PowerMeter Wert, sondern "öffnest" den Inverter. er produziert so viel Strom, wie Solarleistung da ist. So, wie ein nicht smarter Wechselrichter es auch machen würde (ohne Batterie und co). Überschüsse gehen dann ggf. ins Netz.

**Comment by joetheripper** [comment]
Hallo zusammen,

Ich finde frankydee1980's Idee super, da ich vor einem ähnlichen Problem stehe.
Meine Grundlast schwankt bei 120W.
Da bei starker Belastung des Akkus die Spannung sehr schnell einbricht habe ich das obere Leistungslimit auf 200W gesetzt um über die Nacht zu kommen.
Wenn jetzt die Batterie voll und noch überschüssige Sonnenleistung vorhanden ist, würde ich gerne die Obergrenze per MQTT anheben um den Mehrverbrauch (z.B. durch Waschmaschine Kochen oder ähnliches) zu mindern.

Aktuell mache ich das manuell über WebUI aber automatisiert (über MQTT durch meine Hausautomation oder sogar schon auf der DTU) wäre es natürlich am besten.

Ist es möglich das zu implementieren?

Gruß Joe

**Reply by stanpete78** [reply]
Ich hatte meine Antwort zurückgezogen, weil ich deinen Satz falsch gelesen hatte :)
